thirdeye/thirdeye-pinot/config/data/README.md

```diff
@@ -0,0 +1,7 @@
+`ambient_temperature_system_failure.csv`: Hourly data
+ 
+ `Twitter_volumn_AAPL.csv`: 5 Minute data
+ 
+ The above two data file are from https://github.com/numenta/NAB
```

Comment:
Looks like we need to review these licences. LinkedIn doesn't seem to approve them.